package.json

```diff
@@ -17,13 +17,8 @@
   "bugs": {
     "url": "https://github.com/Faire/mjml-react/issues"
   },
-  "main": "dist/src/index.js",
-  "files": [
```

Comment:
We no longer need to specify files. We copy this to the dist folder, and we want to include everything in the dist folder in the publish